### ISSUE TITLE
Overhaul of how EGifts work

### ISSUE THREAD

**Issue by lethality** [PHPfox-Official/phpfox-v4-feature-requests#750]
It's about time this app was adapted to fit into how gamification and egifts work on other platforms, I'm usually critical of the constant attempts to copy facebook, and the like, but look at how the popular games on there handle credits and purchases, look at how another site such as wiredclub handles them...
   My feature request is that to maintain the UX of a gamified App (eGifts + Activity Points), the user should see a front-end price for eGifts that contains the points value, not the converted currency amount. People see money, they don't click, it's as simple as that. It's not good enough to make a user click once or twice to then see he can actually buy it with points.
   Now I'm aware this has been shot down before by @neil96 @ [https://github.com/PHPfox-Official/phpfox-v4-feature-requests/issues/126](https://github.com/PHPfox-Official/phpfox-v4-feature-requests/issues/126)  but I find that reasoning quite odd, and depressing that you'd totally dismiss a better option.

Have you played candy crush or a similar mobile game, what happens when you run out of lives? you can exchange points accrued for extra lives. what does it do if you don't have any points, etc left? It gives you the chance to purchase them with real money.

Look at how wiredclub handles it:
![Initial purchase screen](https://user-images.githubusercontent.com/520295/49491655-f8ca2380-f84c-11e8-8a36-b038658bdfd5.png)

![transaction, points purchase](https://user-images.githubusercontent.com/520295/49491662-fec00480-f84c-11e8-91f9-ca67b853fe83.png)

Activity Points already has the ability to do packages, but nothing really on the front end that uses them, it just makes sense to have everything on the front-end displaying the points required.

![points](https://user-images.githubusercontent.com/520295/49491706-362eb100-f84d-11e8-98a1-63cd0748aad2.png)
Because that ^ looks horrible

**Comment by PhpFoxJohnJr** [comment]
Very nice....some great ideas!

**Comment by prepperz** [comment]
Looks good. Been asking for this for years.

**Comment by juniortimeline** [comment]
Yes, I’ve been waiting this in a year now that’s why I never use eGift on my site because it’s not presentable.
